projects/plugins/jetpack/modules/widgets/gravatar-profile.php

```diff
@@ -274,9 +275,9 @@ public function display_accounts( $accounts = array() ) {
 
 			$sanitized_service_name = $this->get_sanitized_service_name( $account['shortname'] );
 			$link_title             = sprintf(
-				/* translators: %1$s: service username. %2$s: service name ( Facebook, Twitter, etc.) */
+				/* translators: %1$s: account display name. %2$s: service name ( Facebook, Twitter, etc.) */
 				_x( '%1$s on %2$s', '1: User Name, 2: Service Name (Facebook, Twitter, ...)', 'jetpack' ),
```

Comment:
Ideally, we could also replace the `User Name` with something like `Gravatar Display Name`, but since it was referring to the account display name as `User Name`, I think it's fine to keep this name to the Gravatar display name.

The reason I didn't change it was to avoid breaking translations or introducing complexity with a conditional checking for the translation.